androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/components/Checkbuttons.kt

```diff
@@ -1,27 +1,32 @@
 package com.EENX15_22_17.digital_journal.android.ui.components
 
 import androidx.compose.foundation.background
-import androidx.compose.foundation.layout.Box
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.Row
-import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
 import androidx.compose.foundation.shape.CircleShape
 import androidx.compose.material.Checkbox
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.*
 import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.EENX15_22_17.digital_journal.android.ui.theme.background
+import com.EENX15_22_17.digital_journal.android.ui.theme.checkBoxColor
 
 @Composable
 fun <E : Enum<*>> EnumCheckBox(
     choices: Array<E>,
     onSelectionChanged: (selected: Set<E>) -> Unit,
-    currentSelected: MutableSet<E>,
+    currentSelected: Set<E>,
```

Comment:
I would name this `selection` to better convey to the user of the checkbox that its not "Live" data, i.e. The data provided in the constructor is only the initial selection, and the current selection will not be reflected in this variable.

Reply:
Done